Gemfile.lock

```diff
@@ -0,0 +1,216 @@
+GEM
```

Comment:
Yes, I think the .lock file is unnecessary. I will remove it. The Gemfile is something that will make it easier for people to build the site...I can take it out of this commit if it is bothersome. It's not directly related to this PR.